sql/gen-sql-config-docs.py

```diff
@@ -103,6 +103,14 @@ def generate_sql_configs_table_html(sql_configs, path):
                     )
                 )
 
+            if config.name == "spark.sql.files.ignoreInvalidPartitionPaths":
+                description = config.description.replace("<", "&lt;").replace(">", "&gt;")
+            elif config.name == "spark.sql.hive.quoteHiveStructFieldName":
+                description = config.description.replace(
+                    "<", "&lt;").replace(">", "&gt;").replace("`", "&#96;")
```

Comment:
backtick appears in the description of other configs, such as `spark.sql.hive.convertMetastoreInsertDir`
Shoulld it be encoded there as well ?

Reply:
It depends; it’s only for those who need to escape. Well, `convertMetastoreInsertDir` is fine then.